src/PrestaShopBundle/Resources/config/services/core/form/form_data_handler.yml

```diff
@@ -262,3 +261,8 @@ services:
   PrestaShop\PrestaShop\Core\Form\IdentifiableObject\DataHandler\TaxRulesGroupFormDataHandler:
     arguments:
       - '@prestashop.core.command_bus'
+
+  # deprecated
+  prestashop.core.form.identifiable_object.data_handler.feature_form_data_handler:
```

Comment:
Since the feature page was not released yet its related services are still considered experimental, so I think you can remove this definition (especially since there are very few chances it was used directly)